### PR TITLE
[IMP] cooperator: static test helpers

### PR DESCRIPTION
**Pull request by robinkeunen**
Functions in CooperatorTestMixin are often used in setUpClass which is a class method. Helper functions need to be decorared to be accessible.

### CODE REVIEW

File: cooperator/tests/cooperator_test_mixin.py [hunk 1]
```diff
@@ -141,18 +144,23 @@ def get_dummy_subscription_requests_vals(self):
...
             "iban": "BE60096123456870",
             "source": "manual",
         }
 
-    def get_dummy_company_subscription_requests_vals(self):
-        vals = self.get_dummy_subscription_requests_vals()
-        vals.update(
-            {
+        for field, value in custom_vals.items():
+            vals[field] = value
```
Comment (huguesdk) [inline cooperator/tests/cooperator_test_mixin.py]:
```suggestion
        vals.update(custom_vals)
```

File: cooperator/tests/cooperator_test_mixin.py [hunk 2]
```diff
@@ -141,18 +144,23 @@ def get_dummy_subscription_requests_vals(self):
...
+
+        return vals
+
+    @classmethod
+    def get_dummy_company_subscription_requests_vals(cls):
+        vals = cls.get_dummy_subscription_requests_vals(
+            **{
```
Comment (huguesdk) [inline cooperator/tests/cooperator_test_mixin.py]: weird syntax, but clever!

**Review by huguesdk — COMMENTED** [review]
useful change, with some nice improvements, thanks!

File: cooperator/tests/cooperator_test_mixin.py (hunk 2 above)
Comment (robinkeunen) [reply cooperator/tests/cooperator_test_mixin.py]: I actually could use keyword argument just fine ^^

**Comment by robinkeunen** [comment]
@huguesdk I'm leaving the merge to you :-)

**Review by huguesdk — APPROVED** [review]

**Comment by huguesdk** [comment]
@robinkeunen merged (by fast-forward)!

**Pull request merged**